docs/production-deployment.rst

```diff
@@ -191,14 +208,14 @@ to 20::
     helm upgrade anonlink entity-service --namespace=testing --set workers.replicas="20"
 
 
-However, note you may wish to instead keep all configurable values in a yaml file and track
-that in version control.
+However, note you may wish to instead keep all configurable values in a ``yaml`` file and track
+the changes in version control.
 
 Minimal Deployment
 ------------------
 
-To run with minikube for local testing we have provided a ``minimal.yaml`` file that will
-set very small resource limits. Install the minimal system with::
+To run with minikube for local testing we have provided a ``minimal.yaml`` configuration file that will
```

Comment:
where is that `minimal.yaml` file? I couldn't find it in the deployment folder.

Reply:
Hmm, I would have sworn there was one. I'll remove the reference as it was so minimal it couldn't do much except deploy the service